rotkehlchen/db/upgrades/v22_v23.py

```diff
@@ -0,0 +1,20 @@
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from rotkehlchen.db.dbhandler import DBHandler
+
+
+def upgrade_v22_to_v23(db: 'DBHandler') -> None:
```

Comment:
Can you also add a DB upgrade test? It's easy.

You need to open the app with the previous DB version, create an empty account and populate those fields.

Take that DB and put it in the rest of the test DB data. Then use it in a DB upgrade test like done in the rest of the upgrades.

Essentially making sure that the settings are properly migrated to the frontend_settings key. And that the old settings are removed.